File: 03_tuples_arrays/python/arrays.py
"demo of arrays."
from array import array
if __name__ == "__main__":
    xs = array('i', [1, 2, 3, 4, 5])

    # this is dumb, use `numpy.zeros((500,), dtype=numpy.uint64)` instead
    ys = array('Q', [0] * 500)

    print("first element of the array:", xs[0])
    print("second element of the array:", xs[1])

    print("small size:", len(xs))
    print("big size:", len(ys))

    print("a section of the array as a slice", xs[1:3])

    print(xs[5])

File: 03_tuples_arrays/rust/arrays.rs
/// demo of arrays

fn main() {
    let xs = [1, 2, 3, 4, 5];

    let ys: [i32; 500] = [0; 500];
    let _ys = [0_u64; 500];

    println!("first element of the array: {}", xs[0]);
    println!("second element of the array: {}", xs[1]);

    println!("small size: {}", xs.len());
    println!("big size: {}", ys.len());

    println!("borrow a section of the array as a slice {:?}", &xs[1..3]);

    println!("{}", xs[5]);
}
